### DISCUSSION TITLE
How can i make a page ?

### DISCUSSION THREAD

**Discussion by ptqsiscool** [community/community#123186] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

question in title ^^

**Comment by thx-42** [comment]
https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

**Reply by ebndev** [reply]
Hi @ptqsiscool & @theaux42,

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this and related posts.

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
